roles/openshift_node/tasks/main.yml

```diff
@@ -133,11 +133,20 @@
   retries: 120
   delay: 1
   changed_when: false
-  when: openshift.common.is_containerized | bool
 
 - name: Start and enable node
   service: name={{ openshift.common.service_type }}-node enabled=yes state=started
   register: node_start_result
-
+  failed_when: false
+  
+- name: Sleep 10 seconds when starting the node failed
+  pause: secs=10
+  when: node_start_result.state != 'started'
+  
+- name: Try starting the node again
+  service: name={{ openshift.common.service_type }}-node enabled=yes state=started
+  register: node_start_result
+  when: node_start_result.state != 'started'
```

Comment:
I'm thinking of having it be an async task, and then following up on checking the service status using a do-until loop. It might require some tweaking of the delay value to make sure it doesn't check in too early, though.